spec/unit/foreman_external_node_spec.rb

```diff
@@ -51,6 +52,19 @@ class Enc
     expect(webstub).to have_been_requested.times(4)
   end
 
+  it "should NOT connect to the URL in the manifest" do
+
```

Comment:
superfluous newline, but we'll fix that when merging, as I'd like to avoid another hours long test run